src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -783,8 +783,8 @@ private void CheckStake(Block block, ChainedBlock chainTip)
             }
 
             this.logger.LogTrace("Worker #{0} found the kernel.", workersResult.KernelFoundIndex);
-
-            long reward = fees + this.posConsensusValidator.GetProofOfStakeReward(chainTip.Height);
+	        //Get reward for newly created block
```

Comment:
– don't use Tabs, use spaces
– in-line comment should start with space
– comment sentence should and with `.`